test/serverTest.js

```diff
@@ -404,6 +404,16 @@ describe("Running Selenium tests on Chrome Driver", function() {
       });
     });
 
+    it('Checking Jump to Speaker functionality', function(done) {
+      trackPage.jumpToSpeaker().then(function(val) {
+        assert.equal(val, true);
+        trackPage.visit('http://localhost:5000/live/preview/a@a.com/FOSSASIASummit/tracks.html');
```

Comment:
Thanks for the suggestion. Yes, the site is generated locally and then the tests are run on that. The basic idea is that the user sends a PR which contains some changes. We don't want any regressions to come because of that. So, we generate the site incorporating that changes and run it locally itself. We compare the results of the site which is running on the localhost and includes the changes made in the PR and compare it with the expected results. The test then pass or fail according to that :)